File: Day 035/2706. Buy Two Chocolates.py
class Solution(object):
    def buyChoco(self, prices, money):
        prices.sort()
        a = prices[0] + prices[1]
        if a > money:
            return money 
        else:
            return money - a
        """
        :type prices: List[int]
        :type money: int
        :rtype: int
        """

File: Day 035/2706. Buy Two Chocolates.js
/**
 * @param {number[]} prices
 * @param {number} money
 * @return {number}
 */
var buyChoco = function(prices, money) {
    prices.sort((a, b) => a - b);
    const a = prices[0] + prices[1];
    return (a > money) ? money : money - a;
};